vendor/repo_test.go

```diff
@@ -55,7 +55,7 @@ func TestDeduceRemoteRepo(t *testing.T) {
 			url: "https://bitbucket.org/davecheney/hgrepo",
 		},
 		extra: "/cmd/main",
-	}, {
+	}, { */
```

Comment:
I know _you_ won't forget, but just as a public reminder, we do need to figure out what to do about these bitbucket tests. :smile: